effekt/shared/src/main/scala/effekt/Namer.scala

```diff
@@ -288,13 +290,20 @@ object Namer extends Phase[Parsed, NameResolved] {
         resolveGeneric(body)
       }
 
-    case f @ source.ExternDef(capture, id, tparams, vparams, bparams, ret, body) =>
+    case f @ source.ExternDef(capture, id, tparams, vparams, bparams, ret, bodies) =>
+      if (!bodies.supportedByFeatureFlags(Context.compiler.supportedFeatureFlags)) {
+        Context.warning(s"Extern definition ${id} is not supported by this backend.")
+      }
```

Comment:
This is where the warnings are currently generated.

Reply:
Looks great! 👍
For clarity, would it be possible to also dump why exactly it's not supported? (something like `Extern definition ${id} is not supported as it requires feature flags ${featureFlags}, but the current backend only supports ${backendFeatureFlags}`)